liquibase/README.md

```diff
@@ -0,0 +1,166 @@
+# Creating Liquibase snapshots for OpenMRS
+This document explains why Liquibase snapshots are introduced to OpenMRS and how versioning of database changes is 
+affected by snapshots. It also contains instructions for creating new Liquibase snapshots, running them from the 
+console and adding them to the OpenMRS code base.
+
+## Why Liquibase snapshots?
+OpenMRS uses [Liquibase](http://www.liquibase.org/index.html) to evolve its database model. The respective change sets 
+have grown in the course of time and contain unneeded changes. E.g. there are tables or columns that are created, 
+modified and eventually deleted as they are no longer used.
+
+Large and partly outdated Liquibase change sets affect OpenMRS in two ways:
+
+* Unneeded changes slow down the initialisation of OpenMRS
```

Comment:
By changes changes did you mean changesets?

Reply:
Yes, I am happy to change that to "changesets"